src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Locale;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            StringBuilder builder = new StringBuilder();
+            int value = bufferedReader.read();
```

Comment:
let's use readLine() method. Then you can split each line with regexp = \\W+ and check whether or not it starts with letter using only one check: Character.toLowerCase(word.chartAt(0)) == 'w'